media/js/firefox/os/firefox-os-2.5.js

```diff
@@ -8,22 +8,23 @@
     $('.cta-button').on('click', function(e) {
         if (!window._dntEnabled()) {
```

Comment:
Would it be better to put this `if` statement around the event listener? Or am I over-thinking it?